mantidproject/mantidimaging
# Async loading in GUI

Adds async file loading in main window.

Closes #41.

To test:
- Open the GUI and load a large dataset with step set to 1 (i.e. load every image)
- See that the progress bar appears and shows progress (i.e. the bar starts moving fairly soon after clicking load and only full just before loading completes)

(this will be added to other places as appropriate later on, e.g. saving, applying filters, etc)
